osc_sdk/sdk.py

```diff
@@ -520,17 +543,32 @@ def make_request(self, action, *args, **kwargs):
             self.init_ephemeral_auth()
             return self.make_request(action, args, kwargs)
 
-        self.response = self.get_response(res)
+        raw_content = self.get_raw_content(res)
+        self.response = self.get_response(res, raw_content)
+
+    def get_raw_content(self, res):
```

Comment:
I find this comment kind of confusion:
"either with HttpsAdapter or not" imply that this function let caller choose if he want https or not, which should imply another input parameter, which isn't here.
at first look this function simply return content without  HttpsAdapter

Reply:
removed comment